cmake/std/PullRequestLinuxCuda9.2TestingSettings.cmake

```diff
@@ -6,6 +6,8 @@
 
 # Usage: cmake -C PullRequestLinuxCUDA9.2TestingSettings.cmake
 
+set (CMAKE_CXX_STANDARD "14" CACHE STRING "Set C++ standard to C++14")
```

Comment:
Yup, that should do it.

Reply:
@bartlettroscoe the kokkos-kernels changes were made on the develop branch of the repo with SHA kokkos/kokkos-kernels@5ad4fa1, they'll be pulled in the the 3.2.1 patch release (which will clobber the changes in this PR)